packages/api/src/controllers/playback.ts

```diff
@@ -71,6 +73,14 @@ function newPlaybackInfo(
       url: webRtcUrl,
     });
   }
+  if (recordingUrl) {
+    playbackInfo.meta.recordings = [];
```

Comment:
WDYT of calling this something else? We already have an existing "recordings" definitions in our API, which is not what this is. This looks like it returns the URLs of all recordings from this stream, which is not what it does. 

Maybe `historicPlaybackUrl` or `dvrPlayback` or anything else. Doesn't matter if it's maybe a little too verbose IMO since it will mostly be used internally, but it is better than using a misleading name (remember that this will show up in our API reference as well, this is a public-facing API)

Reply:
> (remember that this will show up in our API reference as well, this is a public-facing API)

alternatively, we can move the new field to the `db-schema.yaml` file so it doesn't show up in API reference. That shoul be fine as well